src/main/views/macro/support.njk

```diff
@@ -62,6 +73,9 @@
     {% endif %}
     {{ _emailWidget() }}
     {{ _phoneWidget() }}
+    {% if (toDate() < toDate("2020-01-02") and toDate() > toDate("2019-12-01")) %}
```

Comment:
does this need to be
```suggestion
    {% if (toDate() < toDate("2020-01-02") and toDate() >= toDate("2019-12-01")) %}
```
To go live on the first?

Reply:
done